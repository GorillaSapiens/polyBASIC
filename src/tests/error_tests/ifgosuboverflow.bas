10 IF 5 == 5 GOSUB 10
