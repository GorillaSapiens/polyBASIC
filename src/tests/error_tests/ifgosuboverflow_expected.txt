ERROR: @1:4, GOSUB STACK OVERFLOW
